dev/release/VERIFY.md

```diff
@@ -50,15 +50,28 @@ You can install them by the followings on Debian GNU/Linux and Ubuntu:
 You can install them by the followings on CentOS 7:
 
 ```console
-% sudo yum install -y gobject-introspection-devel
+% sudo yum install -y gobject-introspection-devel git curl
+% cd /etc/pki/rpm-gpg
+% curl -O http://springdale.princeton.edu/data/springdale/7/x86_64/os/RPM-GPG-KEY-springdale
+% cd /etc/yum.repos.d
+% cat << EOF > Springdale-SCL.repo
+[Springdale-SCL]
+name=Springdale - SCL
+baseurl=http://springdale.princeton.edu/data/springdale/SCL/$releasever/$basearch
+gpgcheck=1
+enabled=1
+gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-springdale
+EOF
+% source scl_source enable devtoolset-11
+% source scl_source enable llvm-toolset-13.0
```

Comment:
```suggestion
% scl_source enable devtoolset-11
% scl_source enable llvm-toolset-13.0
```

BTW, can we remove these instructions from `VERIFY.md` because we provide a shell script to setup CentOS 7?